pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
I recall a few people, (@smarterclayton and others) saying this pattern is bad and we should discontinue it. I'm personally fine with the pattern if it actually does error parsing work instead of just wrapping logfatal.

That being said, there is a duplicate declaration of checkErr in pkg/cmd/util/helpers.go (mea culpa; I was lazy and didn't want to sed across all occurrences). IMO, it should just be declared in pkg/cmd/util. Please add a TODO to move it there, and I will do so after merging this.